willow/docker-entrypoint.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+cd /willow 
+
+echo Creating tmp and log
+mkdir -p tmp/pids log
+rm  -f tmp/pids/*
+
+echo "Mounting willow source code as a Docker volume with symlinks, except certain temporary directories and other files"
+
+find ../willow_source/ -maxdepth 1 ! -name "log" ! -name "tmp" ! -name "startup.sh" -exec ln -sf {} ';'
```

Comment:
startup.sh does not exist anymore in this branch. Should the `! -name "startup.sh"` find command exclusion pattern change accordingly to docker-entrypoint.sh?